tests/backends/postgresql/test_operations.py

```diff
@@ -78,3 +79,49 @@ def test_prepare_join_on_clause_different_types(self):
         self.assertEqual(
             rhs_expr, Cast(Col(book_table, book_fk_field), author_id_field)
         )
+
+    def test_bulk_batch_size(self):
+        objects = range(2**16)
+        max_query_params = 2**16 - 1
+        first_name_field = Person._meta.get_field("first_name")
+        last_name_field = Person._meta.get_field("last_name")
+        composite_pk = models.CompositePrimaryKey("first_name", "last_name")
+        composite_pk.fields = [first_name_field, last_name_field]
```

Comment:
I'm starting to wonder if we should hoist these tests out of the oracle & sqlite tests and just have one base test. Then the postgres specific test only needs to test the postgres-specific part where we switch on server-side binding...